src/types/clipboard.ts

```diff
@@ -19,7 +19,7 @@ export interface ClipboardState {
   isPasteAllowed(target: Zone[], clipboardOption?: ClipboardOptions): CommandResult;
 
   paste(target: Zone[], options?: ClipboardOptions | undefined): void;
-  getClipboardContent(): string;
+  getClipboardContent(): Record<string, string>;
```

Comment:
After you defined an enum for the contents (html or plain) of the clipbaord, you can use it to type this (and all the functios that extend it)

```suggestion
  getClipboardContent(): Record<ClipboardContentType, string>;
```